web/wp-config.php

```diff
@@ -66,13 +66,26 @@
  * @link https://github.com/pantheon-systems/quicksilver-examples/blob/master/slack_notification/slack_notification.php
  */
 $wpms_default = array(
-	'WPMS_ON' => false,
 	'WPMS_SMTP_PASS' => 'your_password',
 );
-$secrets = _get_secrets(array('WPMS_ON', 'WPMS_SMTP_PASS'), $wpms_default);
-define( 'WPMS_ON', $secrets['WPMS_ON'] );
+$secrets = _get_secrets(array('WPMS_SMTP_PASS'), $wpms_default);
+
+/**
+ * Configure wp-mail-smtp plugin
+ */
+define( 'WPMS_ON', true );
 define( 'WPMS_SMTP_PASS', $secrets['WPMS_SMTP_PASS'] );
 
+/**
+ * Configure the Sentry integration.
+ */
+define( 'WP_SENTRY_DSN', $secrets['SENTRY_DSN'] );
+define( 'WP_SENTRY_PUBLIC_DSN', $secrets['SENTRY_PUBLIC_DSN'] );
+define( 'WP_SENTRY_ERROR_TYPES', E_ALL & ~E_DEPRECATED & ~E_NOTICE & ~E_USER_DEPRECATED );
```

Comment:
are you sure you want all these error types in sentry?

Reply:
Yeah ... I hear you. This is the configuration referenced in the plugin documentation, but I'm not convinced it is the best one. This might be worth a bit more thought.

Reply:
Note that this is creating a bitmask. I think this is actually probably a reasonable assortment of errors, though I would also include `~E_STRICT`.

Reply:
The full list of available log options is documented here:
http://php.net/manual/en/errorfunc.constants.php

Looking over the list, I think I'm going to re-configure this list a bit. We have a slightly more nuanced set of flags in place on the Drupal side, which I'll bring over to this.

That'll probably land on Monday.